### PR TITLE
`1.20.1` + `1.19.4` release notes

### PR DESCRIPTION
**Pull request by andreafalzetti**
This PR updates the release notes for the Okteto chart `1.20.1` and `1.19.4`.

### CODE REVIEW

File: src/content/release-notes.mdx [hunk 1]
```diff
@@ -5,6 +5,17 @@ sidebar_label: Release notes
...
 
+## 1.20.1
+
+28 May 2024
+
+This version is compatible with Kubernetes versions 1.26 to 1.29
+
+### Improvements
+
+- Updated CLI version in the backend to 2.27.3  <!-- 8135 -->
```
Comment (ifbyol) [inline src/content/release-notes.mdx:16]: Let's elaborate a bit more this indicating why. We could set something like:

`Updated CLI version in the backend to 2.27.3 to fix a panic when using Docker Compose and Okteto Manifests together, and the field image in a compose service was referring to an environment variable`

@codyjlandstrom Any better suggestion?

File: src/content/release-notes.mdx [hunk 2]
```diff
@@ -5,6 +5,17 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.20.1
+
+28 May 2024
+
+This version is compatible with Kubernetes versions 1.26 to 1.29
+
+### Improvements
```
Comment (ifbyol) [inline src/content/release-notes.mdx:14]: It should be a bugfix

Comment (andreafalzetti) [reply src/content/release-notes.mdx]: agree! done ✅

File: src/content/release-notes.mdx (hunk 1 above)
Comment (andreafalzetti) [reply src/content/release-notes.mdx]: I've updated with this suggestion, feel free to update it while I am away

**Review by ifbyol — APPROVED** [review]
It looks good! @codyjlandstrom how do you see it?

File: src/content/release-notes.mdx [hunk 3]
```diff
@@ -5,6 +5,16 @@ sidebar_label: Release notes
...
 
+## 1.20.1
+
+28 May 2024
+
+This version is compatible with Kubernetes versions 1.26 to 1.29
+
+### Bug Fixes
+
+- Updated CLI version in the backend to 2.27.3 to fix a panic when using Docker Compose and Okteto Manifests together, and the field image in a compose service was referring to an environment variable  <!-- 4312, 8135 -->
```
Comment (codyjlandstrom) [inline src/content/release-notes.mdx:16]:
```suggestion
- Updated the backend CLI to version 2.27.3 to resolve a panic issue that occurred when using Docker Compose with Okteto Manifests, specifically when the `image` field in a compose service referred to an environment variable  <!-- 4312, 8135 -->
```

File: src/content/release-notes.mdx [hunk 4]
```diff
@@ -47,6 +57,17 @@ This version is compatible with Kubernetes versions 1.26 to 1.29
...
 
+## 1.19.4
+
+28 May 2024
+
+This version is compatible with Kubernetes versions 1.25 to 1.29
+
+### Bug Fixes
+
+- Updated CLI version in the backend to 2.26.2 to fix a panic when using Docker Compose and Okteto Manifests together, and the field image in a compose service was referring to an environment variable  <!-- 4312, 8137 -->
```
Comment (codyjlandstrom) [inline src/content/release-notes.mdx:68]:
```suggestion
- Updated the backend CLI to version 2.26.2 to resolve a panic issue that occurred when using Docker Compose with Okteto Manifests, specifically when the `image` field in a compose service referred to an environment variable  <!-- 4312, 8137 -->
```

File: versioned_docs/version-1.19/release-notes.mdx [hunk 5]
```diff
@@ -5,6 +5,17 @@ sidebar_label: Release notes
...
 
+## 1.19.4
+
+28 May 2024
+
+This version is compatible with Kubernetes versions 1.25 to 1.29
+
+### Bug Fixes
+
+- Updated CLI version in the backend to 2.26.2 to fix a panic when using Docker Compose and Okteto Manifests together, and the field image in a compose service was referring to an environment variable  <!-- 4312, 8137 -->
```
Comment (codyjlandstrom) [inline versioned_docs/version-1.19/release-notes.mdx:16]:
```suggestion
- Updated the backend CLI to version 2.26.2 to resolve a panic issue that occurred when using Docker Compose with Okteto Manifests, specifically when the `image` field in a compose service referred to an environment variable  <!-- 4312, 8137 -->
```

File: versioned_docs/version-1.20/release-notes.mdx [hunk 6]
```diff
@@ -5,6 +5,17 @@ sidebar_label: Release notes
...
 
+## 1.20.1
+
+28 May 2024
+
+This version is compatible with Kubernetes versions 1.26 to 1.29
+
+### Bug Fixes
+
+- Updated CLI version in the backend to 2.27.3 to fix a panic when using Docker Compose and Okteto Manifests together, and the field image in a compose service was referring to an environment variable  <!-- 4312, 8135 -->
```
Comment (codyjlandstrom) [inline versioned_docs/version-1.20/release-notes.mdx:16]:
```suggestion
- Updated the backend CLI to version 2.27.3 to resolve a panic issue that occurred when using Docker Compose with Okteto Manifests, specifically when the `image` field in a compose service referred to an environment variable  <!-- 4312, 8135 -->
```

File: versioned_docs/version-1.20/release-notes.mdx [hunk 7]
```diff
@@ -46,6 +57,16 @@ This version is compatible with Kubernetes versions 1.26 to 1.29
...
 
+## 1.19.4
+
+28 May 2024
+
+This version is compatible with Kubernetes versions 1.25 to 1.29
+
+### Bug Fixes
+
+- Updated CLI version in the backend to 2.26.2 to fix a panic when using Docker Compose and Okteto Manifests together, and the field image in a compose service was referring to an environment variable  <!-- 4312, 8137 -->
```
Comment (codyjlandstrom) [inline versioned_docs/version-1.20/release-notes.mdx:68]:
```suggestion
- Updated the backend CLI to version 2.26.2 to resolve a panic issue that occurred when using Docker Compose with Okteto Manifests, specifically when the `image` field in a compose service referred to an environment variable  <!-- 4312, 8137 -->
```

**Review by codyjlandstrom — APPROVED** [review]

**Comment by codyjlandstrom** [comment]
> It looks good! @codyjlandstrom how do you see it?

Made some quick copy edits! If you think that sounds ok, it's all good 👍

**Pull request merged**